entity-framework/core/get-started/install/index.md

```diff
@@ -1,110 +1,127 @@
 
 ``` Console
 dotnet add package Microsoft.EntityFrameworkCore.SqlServer
 ```
 
-You can indicate a specific version install in the `dotnet add package` command, using the `-v` modifier. For example, to install EF Core 2.1 packages, append `-v 2.1.0` to the command.
+You can indicate a specific version in the `dotnet add package` command, using the `-v` modifier. For example, to install EF Core 2.1.0 packages, append `-v 2.1.0` to the command.
+
+### Visual Studio NuGet Package Manager UI
+
+* From the menu, select **Project > Manage NuGet Packages**
+
+* Click on the **Browse** or the **Updates** tab
+
+* To install or update the SQL Server provider, select the `Microsoft.EntityFrameworkCore.SqlServer` package, and confirm.
+
+For more information, see [NuGet Package Manager UI](https://docs.microsoft.com/nuget/tools/package-manager-ui).
+
+### Visual Studio NuGet Package Manager Console (PMC)
+
+* From the menu, select **Tools > NuGet Package Manager > Package Manager Console**
+
+* To install the SQL Server provider, run the following command in the PMC:
+
+  ``` PowerShell  
+  Install-Package Microsoft.EntityFrameworkCore.SqlServer
+  ```
+* To update the provider, use the `Update-Package` command.
+
+* To specify a specific version, use the `-Version` modifier. For example, to install EF Core 2.1.0 packages, append `-Version 2.1.0` to the commands
+
+For more information, see [Package Manager Console](https://docs.microsoft.com/nuget/tools/package-manager-console).
+
+## Get Entity Framework Core tools
 
-EF Core includes a set of [additional commands for the `dotnet` CLI](../../miscellaneous/cli/dotnet.md), starting with `dotnet ef`. The .NET Core CLI tools for EF Core require a package called `Microsoft.EntityFrameworkCore.Design`. You can add it to the project using:
+Besides the runtime libraries, you can install tools that can perform some EF Core-related tasks in your project at design time. For example, you can create migrations, apply migrations, and create a model based on an existing database.
+
+Two sets of tools are available:
+* The .NET Core [command-line interface (CLI) tools](../../miscellaneous/cli/dotnet.md) can be used on Windows, Linux, or macOS. These commands begin with `dotnet ef`. 
+* The [Package Manager Console (PMC) tools](../../miscellaneous/cli/powershell.md)  run in Visual Studio 2017 on Windows. These commands start with a verb, for example `Add-Migration`, `Update-Database`.
+
+Although you can use the `dotnet ef` commands from the PMC, it's more convenient to use the PMC tools when you're using Visual Studio:
+* They automatically work with the current project selected in the PMC without requiring manually switching directories.  
+* They automatically open files generated by the commands in Visual Studio after the command is completed.
+
+<a name="cli"></a>
+
+### Get the CLI tools
+
+To get the .NET Core CLI tools for EF Core, install the `Microsoft.EntityFrameworkCore.Design` package:
```

Comment:
I would say here that dotnet ef is included in the .NET Core SDK, but in order to enable it you have to install this design-time package in the project.